web_monitoring/html_diff_render.py

```diff
@@ -681,6 +714,21 @@ def fixup_chunks(chunks):
 
     return result
 
+
+def clean_resource_url(element, other):
+    try:
+        match_element = Strict_url_rule.REGEX_RULES[Strict_url_rule.mode].search(element)
+        if match_element:
+            element = element[match_element.end():]
+        match_other = Strict_url_rule.REGEX_RULES[Strict_url_rule.mode].search(other)
+        if match_other:
+            other = other[match_other.end():]
+        return str.__eq__(element, other) or str.__eq__(element.lower(), other.lower())
+    except KeyError:
+        raise KeyError("{} is an invalid strict URL rule."
+                       "".format(Strict_url_rule.mode))
```

Comment:
You could simplify this with an f-string:

```py
raise KeyError(f'{StrictUrlRule.mode} is an invalid strict URL rule.')
```

Reply:
This is how a KeyError is raised in diffing_server.py but your suggestion is cleaner.